contracts/src/v0.8/mocks/VRFCoordinatorV2Mock.sol

```diff
@@ -37,6 +42,8 @@ contract VRFCoordinatorV2Mock is VRFCoordinatorV2Interface {
     uint96 balance;
   }
   mapping(uint64 => Subscription) s_subscriptions; /* subId */ /* subscription */
+  mapping(address => mapping(uint64 => bool)) s_consumer_added; /* consumer */ /* subId */ /* added */
+  mapping(uint64 => address[]) s_consumers; /* subId */ /* consumers */
```

Comment:
I think this should also be checked when requesting random words. Only consumers of the subscription should be able to make requests